typos.toml

```diff
@@ -2,4 +2,4 @@
 extend-exclude = ["*.svg", "*.json"]
 
 [default]
-extend-ignore-re = ['\d+ths', 'CC BY-NC-ND']
+extend-ignore-re = ['\d+ths', 'CC BY-NC-ND', "trUe", "fAlse"]
```

Comment:
Is there a way we can ignore these per line instead of the entire project?

Reply:
I have no clue. I will do some research

Reply:
no I don't think I can. take a look at the [config ref](https://github.com/crate-ci/typos/blob/master/docs/reference.md):

Reply:
`extend-exclude` to exclude the file instead of the pattern?

Reply:
Ah. Not ideal, but it's not worth the effort to block on this.

Reply:
> `extend-exclude` to exclude the file instead of the pattern?

blocking a whole file is not ideal I'd think.